packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -1067,7 +1070,7 @@ export class SankeyChartBase extends React.Component<ISankeyChartProps, ISankeyC
       xCalloutValue: (singleLink.target as SNode).name,
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       yCalloutValue: (singleLink.source as SNode).actualValue! as any as string,
```

Comment:
I think this `yCalloutValue` computation is a bug. This call-out is for a link, and I think this is supposed to show the `unnormalizedValue` like is done within `_onStreamHover`

Reply:
I looked at the `blame` history for this file and I am convinced that this is a bug. So, I pulled out the common properties to a utility called `linkCalloutAttributes`. Doing so helps keep the visuals for this callout consistent and will reduce the likelihood of such bugs in the future.

Reply:
FYI: @AtishayMsft

Reply:
We need to ensure that your changes are backward compatible and does not break anybody upgrading the package without a deprecation notice.

Reply:
This specific instance seems to be a bug and not impacting backward-compatibility. The problem before this specific line change is that if a user employs the mouse vs. the keyboard to navigate the links in the UX, the user would be shown different weights for the links. This change here fixes that issue so that the link weight is correct regardless of how the user navigates the nodes in the UX.